bokeh/models/tiles.py

```diff
@@ -58,6 +58,10 @@ class MercatorTileSource(TileSource):
     resolution (plot_units / pixels) of minimum zoom level of tileset projection. None to auto-compute.
     """)
 
+    wrap_around = Bool(default=True, help="""
+    flag to indicate whether to load tiles outside of world bounds.
```

Comment:
Two comments on this docstring:
1) You're not outside of world bounds :D you've just looped round
2) Can you add a note about the limitation regarding the axes, so at least its documented somewhere that displaying the coordinates is not expected to work.